techniques/systemSettings/security/fileAlterationMonitoring/2.1/metadata.xml

```diff
@@ -42,6 +42,21 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
         <NAME>MONITOR_ENTITY_NAME</NAME>
         <DESCRIPTION>Path to the file or directory to monitor</DESCRIPTION>
       </INPUT>
+      <SELECT1>
+        <NAME>MONITOR_NOFILE_BEHAVIOR</NAME>
```

Comment:
I would prefer something boolean, like MONITOR_ERROR_NOFILE, or something like this.